conans/client/recorder/action_recorder.py

```diff
@@ -117,17 +125,24 @@ def get_info(self):
         return self.get_install_info()
 
     def get_install_info(self):
-        ret = {"error": self.install_errored,
+        ret = {"error": self.install_errored or self.error,
                "installed": []}
 
-        def get_doc_for_ref(the_ref, the_action):
-            error = None if the_action.type != INSTALL_ERROR else the_action.doc
+        def get_doc_for_ref(the_ref, the_actions):
+            errors = [action.doc for action in the_actions if action.type == INSTALL_ERROR]
+            error = None if not errors else errors[0]
```

Comment:
Why not let the user know about every error:

```python
errors = {action.name: action.doc for action in the_actions if action.type in [...error types...]}
```

and the output will show a dict with all the errors.

Reply:
I think the error is and should be unique here, when an error is recorded it raises an exception.
Even though, there is no `action.name` and with a dict you will loose the order so it should be a list of dicts.
Anyway I will leave it as it is until someone requires something different, currently I think it won't make sense.